### PR TITLE
Add limitation of host params in Ansible playbook jobs

### PR DESCRIPTION
**Pull request by Lennonka**
Requested by https://issues.redhat.com/browse/SAT-25285

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [X] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Comment by Lennonka** [comment]
@adamruzicka I'm wondering if this limitation applies more broadly, not just to Ansible playbooks, but to any/all REX jobs, doesn't it?

File: guides/common/modules/proc_running-an-ansible-playbook.adoc [hunk 1]
```diff
@@ -3,6 +3,11 @@
 
 You can run an Ansible playbook on a host or host group by executing a remote job in {Project}.
 
+.Limitation of host parameters in Ansible playbook job templates
+When you execute an Ansible playbook on multiple hosts and the playbook job template includes a host parameter, {Project} renders the playbook for all hosts in the batch, but only uses the playbook of the first host to execute it on all hosts in the batch.
+Therefore, you cannot use a host parameter to modify the behavior of the playbook per host.
+For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2282275[BZ#2282275].
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-an-ansible-playbook.adoc:9]:
```suggestion
ifndef::orcharhino[]
For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2282275[BZ#2282275].
endif::[]
```

**Comment by adamruzicka** [comment]
> I'm wondering if this limitation applies more broadly, not just to Ansible playbooks, but to any/all REX jobs, doesn't it?

No, this is ansible specific as ansible is natively "run one thing against many targets", whereas script (be it push or pull) is strictly one-to-one

**Review by adamruzicka — APPROVED** [review]

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
Cherry-picked:
- 63dd4be2b8..07648341a4  3.11 -> 3.11
- d110da0790..4807ccd1d7  3.10 -> 3.10
- 299af50d33..3455af2e30  3.9 -> 3.9
- 71921f2cd9..ac42b43499  3.8 -> 3.8
- 82b6445aa9..b9a0d11fbd  3.7 -> 3.7
- 3cf3ffde41..03732a01dd  3.6 -> 3.6
- 6787373d09..8c31fab1ac  3.5 -> 3.5

**Pull request merged**